content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -267,3 +280,53 @@ $ kubectl delete -f $HOME/istio-remote.yaml
 {{< text bash >}}
 $ helm delete --purge istio-remote
 {{< /text >}}
+
+## Stable access to Istio control plane services
+In the above procedure, endpoint IPs of Istio services are gathered and used to
+invoke Helm that will create Istio services on the remote clusters. This updates
+kube-dns in the remote clusters so it can be used to resolve the Istio service names.
+Since Kubernetes pods don't have stable IPs, restart of any Istio service pod in
+the control plane cluster will cause its endpoint to be changed. Therefore, any
+connection made from remote clusters to that endpoint will be broken. This is 
+documented in [Pod restart issue](https://github.com/istio/istio/issues/4822)
```

Comment:
How about update the link as https://github.com/istio/istio/issues/4822#issuecomment-381302100

Reply:
sure will take care of it